src/cljdoc/server/pedestal.clj

```diff
@@ -278,6 +284,23 @@
                                         :body (error/not-found-404)})
               context))})
 
+(defn build-sitemap
+  "Build a new sitemap if previous one was built longer than 60 minutes ago."
+  [{:keys [last-generated sitemap] :as state} storage]
+  (let [now (java.util.Date.)]
+    (if (or (not last-generated)
+            (> (- (.getTime now) (.getTime last-generated)) 360000))
```

Comment:
I think we need an extra 0 in the last-generated check here? If I'm reading this correctly, 360,000ms is 6 minutes.

```suggestion
            (> (- (.getTime now) (.getTime last-generated)) (* 60 60 1000)))
```

Reply:
https://github.com/cljdoc/cljdoc/commit/25c19900b38bdba1d9d364185bace0090ed8cfb6

Reply:
Whoops right. Missed that one :P Thanks for fixing it!